client/app/bundles/course/assessment/containers/AssessmentForm/index.jsx

```diff
@@ -206,15 +206,26 @@ class AssessmentForm extends React.Component {
     const { submitting } = this.props;
     if (this.props.autograded) {
       return (
-        <Field
-          name="skippable"
-          component={Toggle}
-          parse={Boolean}
-          label={<FormattedMessage {...translations.skippable} />}
-          labelPosition="right"
-          style={styles.toggle}
-          disabled={submitting}
-        />
+        <React.Fragment>
```

Comment:
Can use the new short syntax for this. See https://github.com/Coursemology/coursemology2/pull/3084/commits/8b7f00c557fe0709036ba81fff7e9285224b71cd and https://reactjs.org/docs/fragments.html#short-syntax